app/src/main/java/com/bpn/trainme/data/local/TypeConverters.kt

```diff
@@ -0,0 +1,19 @@
+package com.bpn.trainme.data.local
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+class Converters {
+
+    @TypeConverter
+    fun fromList(list: List<String>): String {
+        return Gson().toJson(list)
```

Comment:
Perhaps it would be beneficial to use a more modern library for serialization (Gson is deprecated).

Take a look into kotlinx.serialization and moshi.